.github/workflows/quay-tag.yml

```
@@ -0,0 +1,38 @@
+name: quay.tag
+run-name: Creating new tag in quay based on pushed tag in Github.
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  copy-tag-to-quay:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
+      - name: Install skopeo
+        shell: bash
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install skopeo
+      - name: Login to quay.io
+        shell: bash
+        env:
+          QUAY_TOKEN: ${{ secrets.QUAY_ROBOT_TOKEN }}
+        run: |
+         skopeo login quay.io -u opendatahub+odh_dashboard -p ${QUAY_TOKEN}
```

Can you  move the username to a variable to make it easier to change the bot name with having to commit to the repo?

```suggestion
         skopeo login quay.io -u ${QUAY_USERNAME} -p ${QUAY_TOKEN}
```